WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -108,6 +107,31 @@ def show_user_by_wca_id
     show_user(user)
   end
 
+  def show_registrations_by_comp_id
+    competition = Competition.includes(:events).find_by_id(params[:competition_id])
+    if competition
+      registrations = competition.registrations
+
+      # The request may have a Bearer token, check it
+      current_resource_owner = User.find_by_id(doorkeeper_token&.resource_owner_id)
+      can_manage_competition = current_resource_owner&.can_manage_competition?(competition) || false
```

Comment:
I'm surprised to see the `|| false` here, `nil` is "falsey" in Ruby, so I don't believe this is necessary.